### PR TITLE
feat(ecs): Add artifact support, operation mappings to reference page

### PR DESCRIPTION
**Pull request by allisaurus**
Artifact support added in 1.15.0:  
https://github.com/spinnaker/clouddriver/pull/3825

### CODE REVIEW

**Comment by allisaurus** [comment]
CC: @clareliguori

File: reference/providers/ecs.md
Comment (clareliguori) [inline reference/providers/ecs.md]: Probably someone like @dorbin can help us with correct casing here.  This change has "Resource mapping" and "Operation mapping", but "Server Group" (capital G)

File: reference/providers/ecs.md
Comment (clareliguori) [inline reference/providers/ecs.md]: Double check on ECS -> Amazon ECS everywhere, to match our naming in the official AWS docs

File: reference/providers/ecs.md
Comment (clareliguori) [inline reference/providers/ecs.md]: Should service be upper case or lower case?

File: reference/providers/ecs.md
Comment (clareliguori) [inline reference/providers/ecs.md]: Should we continue doing "instances (tasks)" here and below in Resize?

File: reference/providers/ecs.md
Comment (dorbin) [reply reference/providers/ecs.md]: Sadly, we've got inconsistent capitalization of "server group" on the site. I don't see it as a proper noun, and we're trying to keep our headings to sentence case. So I think it should be "Server group"

Comment (allisaurus) [reply reference/providers/ecs.md]: In [several](https://www.spinnaker.io/reference/providers/cf/) [other](https://www.spinnaker.io/reference/providers/kubernetes/) [reference](https://www.spinnaker.io/reference/providers/appengine/) pages, section headers (e.g., "Resource mapping") are cased as above. "Server Group" is more inconsistent across these same pages, however is either always or mostly capitalized on both words. 

I'll change "Server Group" to always be upper case, but defer to guidance from @dorbin  or others.

File: reference/providers/ecs.md
Comment (allisaurus) [reply reference/providers/ecs.md]: for consistency, ECS resource types seem to be always lower case; will change.

**Review by clareliguori — APPROVED** [review]

**Comment by clareliguori** [comment]
@dorbin Now that 1.15 is out, any feedback before getting this merged?

**Comment by dorbin** [comment]
Sorry, Clare, et al, I was out at a summit all last week.

**Pull request merged**
